runtime/include/chpl-comm-compiler-macros.h

```diff
@@ -53,9 +55,9 @@ void chpl_gen_comm_get(void *addr, c_nodeid_t node, void* raddr,
 #endif
   } else {
 #ifdef CHPL_TASK_COMM_GET
-    chpl_task_comm_get(addr, node, raddr, size, typeIndex, ln, fn);
+    chpl_task_comm_get(addr, node, raddr, size, typeIndex, commID, ln, fn);
```

Comment:
I think you need to do the chpl_cache_comm_*() interface functions as well (above).